src/components/List/index.js

```diff
@@ -35,21 +36,22 @@ export default function List (props) {
   }
 
   const filteredData = type === 'all' ? data : data.filter(listing => typeConvert[listing.type] === type);
-  const filteredDataByCategory = category === 'All' ? filteredData : filteredData.filter(listing => listing.category === category);
+  const filteredDataByCategory = category === 'all' ? filteredData : filteredData.filter(listing => listing.category === category);
```

Comment:
These `'all'` strings would communicate more clearly if they used a defined constant. (`type` and `category`). It's a little unclear from this small code snapshop if they reference the same thing, what other values there might be, etc.

Reply:
I've done this, there are more changes I'd like to look at with importing the config file where it's needed instead of passing it down as props - I'll look into that later.